pytest/conftest.py

```diff
@@ -78,7 +78,7 @@ def base_sample_resource(username='admin', title=str(uuid.uuid4()), contributor=
         owner=user,
         title=title,
         metadata=metadata,
-        files=(open('pytest.ini', 'rb'),)
+        # files=(open('pytest.ini', 'rb'),)
     )
```

Comment:
This varies on filesystems and is unused, anyway. To make it more portable for now, until this is generalized, it's commented out. Ideally, leave it commented as a stub for people to pick up and work on quicker in the future.